.ocamlformat

```diff
@@ -1,4 +1,4 @@
-version=0.26.1
+version=0.26.2
```

Comment:
Can we hold off on this upgrade until 26.2 is available in nix? We could make an override for ocamlformat, but it doesn't seem like it's worth it.

Reply:
I dropped it in my last rebase.